src/main/java/com/google/sps/servlets/NewsServlet.java

```diff
@@ -0,0 +1,49 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import com.google.gson.Gson;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.sps.data.Article;
+import com.google.sps.data.ArticleBuilder;
+
+/**
+ * Servlet for generating news articles from keywords.
+ */
+@WebServlet("/news")
+public class NewsServlet extends HttpServlet {
+
+  private List<String> keywords = new ArrayList<String>();
```

Comment:
[Optional] You technically don't need the second <String> generic type definition here. For example:
`List<String> keywords = new ArrayList<>();`
also works, and does the same thing.